src/core/trulens/core/app/base.py

```diff
@@ -1400,29 +1400,38 @@ def _handle_record(
         if len(self.feedbacks) == 0:
             return []
 
-        # Add empty (to run) feedback to db.
```

Comment:
Can you add some documentation to what is happening in this code like the line that was removed?

Reply:
Done.